taui/src/components/select-account.js

```diff
@@ -0,0 +1,208 @@
+// @flow
+import Storage from '@aws-amplify/storage'
+import message from '@conveyal/woonerf/message'
+import React from 'react'
+
+import type {AccountProfile} from '../types'
+
+/**
+ * Search and select from accounts on S3.
+ */
+export default class SelectAccount extends React.PureComponent<Props> {
+  state = {
+    accounts: [],
+    componentError: null,
+    errorMessage: '',
+    headOfHousehold: '',
+    voucherNumber: ''
+  }
+
+  constructor (props) {
+    super(props)
+
+    this.changeHeadOfHousehold = this.changeHeadOfHousehold.bind(this)
+    this.changeVoucherNumber = this.changeVoucherNumber.bind(this)
+    this.createAccount = this.createAccount.bind(this)
+    this.deleteAccount = this.deleteAccount.bind(this)
+    this.selectAccount = this.selectAccount.bind(this)
+    this.search = this.search.bind(this)
+  }
+
+  changeHeadOfHousehold (event) {
+    this.setState({headOfHousehold: event.target.value})
+    this.setState({errorMessage: ''})
+  }
+
+  changeVoucherNumber (event) {
+    this.setState({'voucherNumber': event.target.value})
+    this.setState({errorMessage: ''})
+  }
+
+  createAccount () {
+    const search = this.search
+    const name = this.state.headOfHousehold
+    const voucher = this.state.voucherNumber
+
+    if (!name || !voucher) {
+      console.error('Missing name or voucher')
+      this.setState({errorMessage:
+        'Enter both name and voucher to create account.'})
+      return
+    } else {
+      this.setState({errorMessage: ''})
+    }
+
+    const key = name.toUpperCase() + '_' + voucher.toUpperCase()
+    console.log('Creating account profile for ' + key)
```

Comment:
There are a few `console.log` statements in this file that look like they're mostly for debugging purposes. If you're planning to use them in future work I think it's fine for them to stick around for the moment but I wanted to flag them to make sure we know they exist and will need to be removed eventually.

Reply:
I made a pass to clear out extraneous console logging outside of this file, but yes, there's some partially-implementing things here I intend to clear of logging with the next task.